tracee-ebpf/tracee/pipeline.go

```diff
@@ -110,13 +112,14 @@ func (t *Tracee) processRawEvent(done <-chan struct{}, in <-chan RawEvent) (<-ch
 }
 
 func (t *Tracee) getStackAddresses(StackID uint32) ([]uint64, error) {
+	StackIDC := C.uint(StackID)
 	StackAddresses := make([]uint64, maxStackDepth)
 	stackFrameSize := (strconv.IntSize / 8)
 
 	// Lookup the StackID in the map
 	// The ID could have aged out of the Map, as it only holds a finite number of
 	// Stack IDs in it's Map
-	stackBytes, err := t.StackAddressesMap.GetValue(StackID)
+	stackBytes, err := t.StackAddressesMap.GetValue(unsafe.Pointer(&StackIDC))
```

Comment:
I think it should be ok if you just give `unsafe.Pointer(&StackID)` and remove the "import C" part.